### PR TITLE
feat: align amvf config with athena

### PR DESCRIPTION
**Pull request by andiwand**
config take from https://gitlab.cern.ch/atlas/athena/-/blob/master/Tracking/Acts/ActsTrkTools/ActsTrkPriVtxFinderTool/src/AdaptiveMultiPriVtxFinderTool.h

### CODE REVIEW

**Comment by andiwand** [comment]
I guess this is bad because the unit tests couple to the default values. might be better to overwrite the options in the examples?

File: Core/include/Acts/Utilities/AnnealingUtility.hpp [hunk 1]
```diff
@@ -30,8 +30,7 @@ class AnnealingUtility {
   /// @brief The configuration struct
   struct Config {
     // Config constructor with default temperature list: {64.,16.,4.,2.,1.5,1.}
```
Comment (AJPfleger) [inline Core/include/Acts/Utilities/AnnealingUtility.hpp:32]: Should we update the comment too?
```suggestion
    // Config constructor with default temperature list: {1.}
```

**Review by AJPfleger — COMMENTED** [review]
lgtm. I also rechecked the values (y)

**Comment by andiwand** [comment]
@AJPfleger could you take another look?

**Pull request merged**